src/components/map/MapService.js

```diff
@@ -1503,12 +1508,15 @@ goog.require('ga_urlutils_service');
             }
           }).then(function(response) {
             var height = carto.height - response.data.height;
-            var offset = Math.tan(Cesium.Math.toRadians(50)) * (height + 50);
-            destination.x += offset * 2;
+            var offsetView = Math.tan(
+                Cesium.Math.toRadians(pitch + cameraFieldOfView / 2)) * height;
```

Comment:
because you want to be in the middle of the camera field view. Need drawing tomorrow.